source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -40,11 +40,30 @@ values from resolv.conf will be used. If either of them is used, it
 will be passed to VDSM overriding configuration from resolv.conf. For
 example we can set DNS configuration in a Network related to certain DC
 (or later to certain Cluster) and this configuration will override
-configuration in resolv.conf. If we now setup configuration in
+configuration in resolv.conf. Note: when Network is being attached in 
+certain host, NetworkAttachment record is created for this, and 
+dns configuration is copied from Network record to NetworkAttachment 
+record. There we can further modify it. If we now setup configuration in
```

Comment:
What happens when the cluster-wide configuration changes? It would be nice if all hosts are updates (similarly to a change of vlan or label), except for hosts that have a local override.

I think that it would be better if we can avoid this copy until someone wants to override the cluster-wide configuration. 

Please add a discussion about what happens when we change DNS config in the network while it is already the management network of multiple hosts.